sdk/ai/azure-ai-agents/samples/agents_async/sample_agents_deep_research_async.py

```diff
@@ -42,51 +43,171 @@
 from azure.identity.aio import DefaultAzureCredential
 
 
+def convert_citations_to_superscript(markdown_content):
+    """
+    Convert citation markers in markdown content to HTML superscript format.
+
+    This function finds citation patterns like 【78:12†source】 and converts them to
```

Comment:
Let us avoid Unicode characters in the comments, it will break in some IDE-s. May be we can say
"This function finds citation patterns, limited by Unicode brackets, for example '\u301078:12 source\u3011'"